src/lucky/params.cr

```diff
@@ -143,6 +143,25 @@ class Lucky::Params
     nested_file_params(nested_key.to_s)
   end
 
+  # Converts the params in to a `Hash(String, String)`
+  #
+  # ```crystal
+  # params.to_h  # Hash(String, String)
+  # ```
+  def to_h
+    hash = {} of String => String | Hash(String, String)
+    params = body_params.to_h.merge(query_params.to_h)
+    params.map do |key, value|
+      matches = key.split(':')
+      if matches.size > 1
```

Comment:
Maybe extract local var called `is_nested = matches.size > 1` so it is easier to tell what the conditional is?